src/Sentry.NLog/SentryTarget.cs

```diff
@@ -345,6 +355,21 @@ protected override void Write(LogEventInfo logEvent)
             }
         }
 
+        private User GetUser(LogEventInfo logEvent)
+        {
+            if (User == null)
+                return null;
```

Comment:
Please add `{}` to keep consitency with the code-base. 
If there's a flag for `editorconfig` it would be nice to get that on